.npmrc

```diff
@@ -1 +1 @@
-@biigle:registry=https://npm.pkg.github.com
+@biigle:registry=https://npm.pkg.github.com:_authToken=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
```

Comment:
:warning: You published your secret GitHub access token. You should revoke the token immediately.

Reply:
Yes big mistake from my site.